### PR TITLE
Adding support for 'tab to pick' [related to #19]

### PR DESCRIPTION
**Pull request by oleander**
This pr implements #19.

### CODE REVIEW

File: anycomplete.lua [hunk 1]
```diff
@@ -4,13 +4,34 @@ local mod = {}
...
+        chooser:choices({})
+    end
+
+    tab = hs.hotkey.bind('', 'tab', function()
+        local id = chooser:selectedRow()
+        local item = choices[id]
+        -- If no row is selected, but tab was pressed
+        if not item then return end
+        chooser:query(item.text)
+        reset()
```
Comment (nathancahill) [inline anycomplete.lua]: I don't think `reset` is needed here.

**Comment by nathancahill** [comment]
Looks great, thanks!

**Comment by oleander** [comment]
I'll merge this with master, hold on.

**Comment by oleander** [comment]
Merged.

**Comment by nathancahill** [comment]
Thanks!

**Pull request merged**